src/main/java/pro/sky/telegrambot/listener/TelegramBotUpdatesListener.java

```diff
@@ -0,0 +1,90 @@
+package pro.sky.telegrambot.listener;
+
+import com.pengrad.telegrambot.TelegramBot;
+import com.pengrad.telegrambot.UpdatesListener;
+import com.pengrad.telegrambot.model.Update;
+import com.pengrad.telegrambot.request.SendMessage;
+import com.pengrad.telegrambot.response.SendResponse;
+import pro.sky.telegrambot.model.NofiticationTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import pro.sky.telegrambot.repository.NofiticationRepository;
+import pro.sky.telegrambot.servise.NofiticationServise;
+
+import javax.annotation.PostConstruct;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Service
+public class TelegramBotUpdatesListener implements UpdatesListener {
+
+    private Logger logger = LoggerFactory.getLogger(TelegramBotUpdatesListener.class);
+
+
+
+
+    @Autowired
+    private TelegramBot telegramBot;
+    private NofiticationServise nofiticationServise;
```

Comment:
Чтобы NotificationServise (NotificationServi**c**e) появился, спринг должен его заинжектить. Просто так он это дело не будет, ему нужно сказать, что это нужно сделать. Для этого нужно либо конструктор создать, либо нужные свойства пометить аннотацией Autowired
```suggestion
    @Autowired
    private NofiticationServise nofiticationServise;
```